hive-tables-creation.py

```diff
@@ -1,6 +1,12 @@
 import boto3
 import csv
 import logging
+import logger
+
+level = "info"
+logger_path = "/var/log/adg/hive_tables_creation_log.txt"
+logger_format = "{ 'timestamp': '%(asctime)s', 'log_level': '%(levelname)s', 'message': '%(message)s' }"
+the_logger = setup_logging(level, logger_path , logger_format)
```

Comment:
didnt you move this to the logger.py already?

Reply:
This preparing variable for the function call and calling the function